tests/test_utils.py

```diff
@@ -100,13 +100,23 @@ def test_calculate_job_input_hash():
     ) == calculate_job_input_hash(job_spec_2, workflow_json)
 
 
-def test_calculate_file_access_time(sample_workflow_workspace):  # noqa: F811
+def test_calculate_file_access_time(tmp_path):
     """Test calculate_file_access_time."""
-    sample_workflow_workspace_path = next(sample_workflow_workspace("sample"))
-    access_times = calculate_file_access_time(sample_workflow_workspace_path)
-    all_file_paths = list(Path(sample_workflow_workspace_path).rglob("*.*"))
-    for file_path in all_file_paths:
-        assert str(file_path) in access_times
+    before_writing_files = time.time() - 1
+    tmp_path.joinpath("a.txt").write_text("content of a")
+    tmp_path.joinpath("subdir").mkdir()
+    tmp_path.joinpath("subdir", "b.txt").write_text("content of b")
+    tmp_path.joinpath("c.txt").symlink_to("a.txt")
+    tmp_path.joinpath("another_subdir").mkdir()
+    before_calculating_times = time.time() + 1
```

Comment:
Just a very minor note: I found the name of the variable together with the `+1` a bit confusing, since because of that `+1` it will probably contain a time which is bigger than the time at which the times are calculated (therefore being "after" calculating times).
I guess the goal is checking that the access times are correctly returned and this means that they will need to be "later" then `before_writing_files` and "earlier" than the moment in which the times are calculated, right? And I guess that the `-1` and the `+1` are rightly added to account for any potential delays in file system operations.

If so, what about renaming the variable to `after_writing_files` to remove the ambiguity?
It's in any case a triviality, so feel free to amend it before merging or leave it as is :)